### PR TITLE
[SET-403] add default value and var if specified to run as build script

### PR DESCRIPTION
**Pull request by baranowb**
Issue: https://issues.redhat.com/browse/SET-403
related: https://github.com/jboss-set/cedalion/pull/7

### CODE REVIEW

**Comment by rpelisse** [comment]
LGTM!

@spyrkob we should add a JVM_MEMORY_XML envVar (or something like that in both eap-job*.sh, but let's table that for now)

File: eap-job-64.sh [hunk 1]
```diff
@@ -221,7 +221,7 @@ else
   export TESTSUITE_OPTS="${TESTSUITE_OPTS} -Djboss.test.mixed.domain.dir=${OLD_RELEASES_FOLDER}"
   export TESTSUITE_OPTS="${TESTSUITE_OPTS} -Dmaven.test.failure.ignore=${MAVEN_IGNORE_TEST_FAILURE}"
   export TESTSUITE_OPTS="${TESTSUITE_OPTS} -Dsurefire.rerunFailingTestsCount=${RERUN_FAILING_TESTS}"
-  export TESTSUITE_OPTS="${TESTSUITE_OPTS} -Dsurefire.memory.args=-Xmx1024m"
+  export TESTSUITE_OPTS="${TESTSUITE_OPTS} -Dsurefire.memory.args=-Xmx4048m"
```
Comment (TomasHofman) [inline eap-job-64.sh:224]: @baranowb is the `-Xmx4048m` important for anything? EAP 7 script uses 1024 too...

Comment (TomasHofman) [reply eap-job-64.sh:224]: But if it works I'm OK with it.

Comment (baranowb) [reply eap-job-64.sh:224]: failed to init JVM with less. Something is screwing with memory.

Comment (TomasHofman) [reply eap-job-64.sh:224]: Do you have a link that shows that problem? Might have been caused by too large Xms for instance.

Comment (baranowb) [reply eap-job-64.sh:224]: Nope, rolling log erased those. AFAIR it did say Xms was too big. Bit it never was bigger than 1G. So its something wonky I really dont want to look into :)

**Comment by TomasHofman** [comment]
LGTM

File: eap-job-64.sh (hunk 1 above)
Comment (baranowb) [reply eap-job-64.sh:224]: "Initial Heap Size Set To A Larger Value Than The Maximum Heap Size" was the msg AFAIR.

**Pull request merged**